### ISSUE TITLE
Still Having This Error: editor_resource_preview.cpp(361) : error C4716: 'EditorResourcePreview::check_for_invalidation': must return a value

### ISSUE THREAD

**Issue by ranmaru90** [godotengine/godot#5578]
**Operating system or device - Godot version:**
Windows 10 Pro. Latest compiled from the GitHub repo. 

**Issue description** (what happened, and what was expected):
When compiling with Microsoft's Visual C++ compiler (both 32-bit and 64-bit), compilation fails with error 

```
godot\tools\editor\editor_resource_preview.cpp(361) : error C4716: 'EditorResourcePreview::check_for_invalidation': must return a value
scons: *** [tools\editor\editor_resource_preview.windows.opt.tools.64.obj] Error 2
scons: building terminated because of errors.
```

**Steps to reproduce:**

1.  Pull the latest source from GitHub.
2. Try to compile.

**Comment by vnen** [comment]
Should be already fixed by 7c1ab42571d5291952e329554643142a30dee1d1.

**Comment by akien-mga** [comment]
That was a good example of bad team communication :)
